checkout/payment-menu.md

```diff
@@ -301,7 +307,7 @@ finalizing the payment in the [After Payment section][after-payment].
 [operations]: /checkout/other-features#operations
 [order-items]: #order-items
 [payee-reference]: /checkout/other-features#payeereference
-[payment-menu-image]: /assets/img/checkout/payment-menu.png
+[login-payment-menu-image]: /assets/img/checkout/logged-in-payment-menu-450x900.png
```

Comment:
Tip: Use command palette (Ctrl+Shift+P) and the "Sort Lines Ascending" to sort the links alphabetically.